sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/ied/IEDAdapterTest.java

```diff
@@ -226,4 +226,17 @@ void createControlBlock() throws Exception {
         controlBlock.setHolderIEDName("IED_NAME");
         controlBlock.setHolderIEDName("IED_NAME");
     }
+
+    @Test
+    void addPrivate() throws Exception {
+        SCL scd = SclTestMarshaller.getSCLFromFile("/ied-test-schema-conf/ied_unit_test.xml");
```

Comment:
You can also mock your class like this, instead of working in a complete file
```suggestion
        
        SclRootAdapter sclRootAdapter = Mockito.mock(SclRootAdapter.class);
        SCL scl = Mockito.mock(SCL.class);
        Mockito.when(sclRootAdapter.getCurrentElem()).thenReturn(scl);
        TIED tied = new TIED();
        Mockito.when(scl.getIED()).thenReturn(List.of(tied));
```